backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/exportimport/ImportVmFromExternalProviderCommand.java

```diff
@@ -189,17 +202,39 @@ protected boolean validate() {
 
         if (getVm().getOrigin() != OriginType.KVM &&
                 getParameters().getVirtioIsoName() != null &&
+                getVirtioIsoStorageDomainType() == StorageDomainType.ISO &&
```

Comment:
Cool, does this fix, together with the other changes here, https://bugzilla.redhat.com/2109021 ?

Reply:
Yes. As I see, https://bugzilla.redhat.com/2109021 is the same bug as https://bugzilla.redhat.com/1721455.